libmscore/edit.cpp

```diff
@@ -1053,9 +1053,26 @@ void Score::regroupNotesAndRests(const Fraction& startTick, const Fraction& endT
                                     continue; // this voice is empty here
                               if (!cr->isRest() || s->tick() + cr->actualTicks() > maxTick || toRest(cr)->isGap())
                                     break; // next element in the same voice is not a rest, or it exceeds the selection, or it is a gap
+                              if (cr->tuplet()) {
+                                    Tuplet* tuplet = cr->tuplet(); // element is a tuplet; special case
+                                    short tupletSize = tuplet->elements().size();
```

Comment:
`size_t`, not `short`

Reply:
I believe this results in a compiler warning

Reply:
This line has been removed due to another comment, so this is fixed